### PR TITLE
sensor object getters for the easygopigo3 class

### PR DESCRIPTION
**Pull request by RobertLucian**
I'm referencing this PR: https://github.com/CleoQc/GoPiGo3/pull/1

With this PR, the creation of a specific object is a lot more straightforward and easy.

This requires changing all the example programs accordingly.

### CODE REVIEW

File: Software/Python/easygopigo3.py [hunk 1]
```diff
@@ -293,12 +293,41 @@ def turn_degrees(self, degrees, blocking=False):
...
+    def create_buzzer(self, port):
+        return Buzzer(port, self)
+
+    def create_led(self, port):
+        return Led(port, self)
+
+    def create_button_sensor(self, port):
+        return ButtonSensor(port, self)
+
+    def create_linefollower(self, port):
```
Comment (CleoQc) [inline Software/Python/easygopigo3.py]: to stay standard, could this method be called create_line_follower ?

Comment (CleoQc) [inline Software/Python/easygopigo3.py]: Thinking out loud for now. The use of the verb "create" makes sense from an OOP point of view (we create an instance) but it is totally meaningless from a user point of view. The sensor is there in their hand. They're just setting a port and gaining access to software functionality.
I'm trying to think of a better word and I'm open to suggestions.

Comment (CleoQc) [inline Software/Python/easygopigo3.py]: Ok, after talking a bit with @TarynSullivan  we're going with assign_button_sensor etc..
Just change create for assign

Comment (RobertLucian) [inline Software/Python/easygopigo3.py]: Will go this way.

**Comment by CleoQc** [comment]
changing my request to use init_  instead of assign_ instead of create_  :)

so init_light_sensor(port) etc...

**Pull request merged**